front/src/components/Chat/DirectMessage.tsx

```diff
@@ -92,7 +92,8 @@ function InfoFriend(props: { user: IUser }) {
 	}
 
 	const sendFriendRequest = async () => {
-		await new Promise((resolve) => setTimeout(resolve, 1000));
+		// await new Promise((resolve) => setTimeout(resolve, 1000));
+		// Ca sert a quoi ca ? peut t'on remove ?
```

Comment:
maybe (Honestly I copied this code from Profile header (Friend request button) so I'm not very sure :sweat_smile: :sweat_smile:  )
If it works fine without this line we should remove